Feature/gdh 745 table mobile

robbieatacato opened the pull request:
### Solve:
https://acato-nl.atlassian.net/browse/GDH-745

- bad usability on mobile

### Purpose

- better usability on mobile by adding repsonsive scroll buttons, for horizontal scrolling

### Screen capture 1:

https://user-images.githubusercontent.com/95216123/188323643-ca0fe9fe-8444-4d9a-b3fb-a6de955faaa9.mov

### Screen capture 2:

https://user-images.githubusercontent.com/95216123/188323696-3c7973e7-81da-45a4-9c1c-991de0923a39.mov

### Screen capture 3:

https://user-images.githubusercontent.com/95216123/188323771-7c47031c-1420-4a24-a457-ee7fa49f80b1.mov

Screen capture 4:

https://user-images.githubusercontent.com/95216123/188323952-014aa752-f35f-4ea2-b418-5668802a8ebd.mov

closes #1082

Robbert commented:
Looks like you made a cool improvement for mobile users!

We have some docs on tables on the NL Design System website, especially regarding accessibility: [Table component](https://www.nldesignsystem.nl/docs/componenten/Community/table). Long story short: it is not acceptable for accessibility to display a table without the official HTML table elements, or without the default `display: table` etc.

I hope you are able to make a version that still uses the original HTML tables.

Robbert reviewed (dismissed):
Please consider an approach that is not known to be inaccessible.

robbieatacato commented:
> Looks like you made a cool improvement for mobile users!
> 
> We have some docs on tables on the NL Design System website, especially regarding accessibility: [Table component](https://www.nldesignsystem.nl/docs/componenten/Community/table). Long story short: it is not acceptable for accessibility to display a table without the official HTML table elements, or without the default `display: table` etc.
> 
> I hope you are able to make a version that still uses the original HTML tables.

@Robbert , I did not change the **_html of the table itself_**, but rather added a div wrapper around it, like so:

Things I added in the bem/html:

- `<div class="denhaag-table">`
- `<div class="denhaag-button-group denhaag-button-group--multiple denhaag-table__scroll-buttons">`
- `<div class="denhaag-table__container">`


Things I changed in the bem/html:

- <table class="denhaag-table__table"> : before the css class was named `denhaag-table`

paulAcato reviewed (changes requested):
Rather would see `denhaag-table-wrapper` so the `table` would keep the `denhaag-table` css class.

Robbert commented:
@robbieatacato Mea culpa, I should have paid closer attention to the diff 🙈

robbieatacato commented:
> Rather would see `denhaag-table-wrapper` so the `table` would keep the `denhaag-table` css class.

@paulAcato I refactored the bem classes + css classes + js classes, in the following commit: https://github.com/nl-design-system/denhaag/pull/1082/commits/ba86f665fc43fdf170140b6b3f4c6dd92f6e1869

robbieatacato commented:
@Yolijn Ik heb de tables weer teruggezet + de responsive horizontal scroll variant toegevoegd.
Zie: https://github.com/nl-design-system/denhaag/pull/1082/files

Yolijn reviewed (approved):
Awesome @robbieatacato Nice to have 2 table variants and keep support for a table-caption with it's own styling
